models/ed/R/model2netcdf.ED2.R

```diff
@@ -103,18 +103,18 @@ model2netcdf.ED2 <- function(outdir, sitelat, sitelon, start_date, end_date, pft
                                outdir, start_date, end_date, 
                                pft_names)
     }
-    
+    #length(out_list[["-T-"]]); length(out_list[["-E-"]])
```

Comment:
debugging code?

Reply:
Yes, thank you.  I will remove those before next sync!